### PR TITLE
Minor fixes for typos

### PR DESCRIPTION
**Pull request by quangx**
fix step76 document

### CODE REVIEW

File: examples/step-76/step-76.cc [hunk 1]
```diff
@@ -594,7 +594,7 @@ namespace Euler_DG
 
 
   // The following function does an entire stage of a Runge--Kutta update and is
-  // - alongside the slightly modified setup - the heart of this tutorial
+  // alongside the slightly modified setup - the heart of this tutorial
```
Comment (tjhei) [inline examples/step-76/step-76.cc:597]: This fixes the problem that doxygen formats this as a list, but I think the grammar is a bit awkward now. I would do 
" ... and is, alongside the slightly modified setup, the ..."

**Review by tjhei — COMMENTED** [review]
Thank you, @quangx ! I will get in touch with you and walk you through part 2 of this.

(if you scroll up you will see that you accidentally include a second unrelated commit, we will get rid of that)

**Review by masterleinad — APPROVED** [review]

**Review by bangerth — COMMENTED** [review]
I'll let @tjhei show you how to squash things into a smaller number of commits. But I appreciate you take the time to fix these things!

File: examples/step-79/doc/results.dox [hunk 2]
```diff
@@ -48,8 +48,7 @@ These pictures show that what we find here is in accordance with what one
 typically sees in other publications on the topic @cite Bendse2004. Maybe more interestingly, the
 result looks like a truss bridge (except that we apply the load at the top of
 the trusses, rather than the bottom as in real truss bridges, akin to a "deck
-truss" bridge), suggesting that the designs that have been used in bridge-
-building for centuries are indeed based on ideas we can now show to be optimal
+truss" bridge), suggesting that the designs that have been used in bridge-building for centuries are indeed based on ideas we can now show to be optimal
```
Comment (tjhei) [inline examples/step-79/doc/results.dox:51]:
```suggestion
truss" bridge), suggesting that the designs that have been used in bridge-building
for centuries are indeed based on ideas we can now show to be optimal
```

**Review by drwells — APPROVED** [review]

**Review by tjhei — APPROVED** [review]

**Review by masterleinad — APPROVED** [review]

**Pull request merged**